util/cache.go

```diff
@@ -26,6 +26,8 @@ func (c *Cache) Run(in <-chan Param) {
 		key := p.Key
 		if p.LoadPoint != nil {
 			key = fmt.Sprintf("lp-%d/%s", *p.LoadPoint+1, key)
+		} else if len(p.Meter) > 0 {
```

Comment:
Not quite sure, if I get you right.
Cache won't depend on meters, It just evaluates if a meter name (string) is assigned to this instance of the Param struct. If I do not evaluate this here, it will be treated as "regular" instance of the Param struct.
Or should the aux meters be communicated via a different channel?

Reply:
We cannot have a utility function start interpreting the contents of the message. Doing this for loadpoint (which is at least a central concept) is bad enough. Why is this needed at all instead of publishing the values as-is?

Reply:
Ok, I got your point that passing the loadpoint is already a bad design. Actually, i also believe that injecting an array if all the loadpoints into e.g. influxdb.go:Run() looks inappropriate. 

I have to mention, that I just started learning go when I got into this project. Originally, i am coming from the C++ world. From what i understood is, that go channels (together with the Param struct) is used as a kind of message bus.
My question is: if we pass around a Param struct with only a key and a value (what i guess what you mean with "as-is"), how would someone distinguish between multiple instances of message sources (e.g. loadpoints or meters)?
In other words, what would be a better practice in Go?
Another channel?

Or: how would you realize this with a better design?